/database: fingerprints.pkl
/macaddr_1: "08:60:6e:cc:79:04"
/essid_1: cob-developer
/macaddr_2: "84:c9:b2:6a:80:e8"
/essid_2: cob3-3-extern
/macaddr_3: "84:c9:b2:6a:81:30"
/essid_3: desire-extern 
/macaddr_4: "00:0b:0e:c9:03:83" 
/essid_4: IZS-Campus
/macaddr_5: "00:0b:0e:c9:38:c1"
/essid_5: eduroam
